src/index.html

```diff
@@ -1,13 +1,83 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div
+      class="card-block"
+      data-qa="card"
+    >
+      <img
+        class="card-block__pic"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+
+      <div class="card-block__name">
+        <h2 class="card-block__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div class="card-block__par">
+        <div class="card-block__parag">
```

Comment:
Use `<p>` tag for text
```suggestion
        <p class="card-block__parag">
```

Reply:
same here